src/Features/CSharp/IntroduceVariable/CSharpIntroduceVariableService_IntroduceLocal.cs

```diff
@@ -59,6 +59,11 @@ internal partial class CSharpIntroduceVariableService
                 return Task.FromResult(IntroduceLocalDeclarationIntoLambda(
                     document, expression, newLocalName, declarationStatement, parentLambda, allOccurrences, cancellationToken));
             }
+            else if (IsInExpressionBodiedMember(expression))
```

Comment:
Either way, add some tests for combining expression-bodied members with AnonymousMethodExpressions (above) and lambdas, such as
```C#
    Func<int, int> X() => y => 8;
```